src/main/java/org/sefglobal/scholarx/service/MenteeService.java

```diff
@@ -0,0 +1,40 @@
+package org.sefglobal.scholarx.service;
+
+import org.sefglobal.scholarx.exception.ResourceNotFoundException;
+import org.sefglobal.scholarx.model.Mentee;
+import org.sefglobal.scholarx.repository.MenteeRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+public class MenteeService {
+
+    private final static Logger log = LoggerFactory.getLogger(MenteeService.class);
+    private final MenteeRepository menteeRepository;
+
+    public MenteeService(MenteeRepository menteeRepository) {
+        this.menteeRepository = menteeRepository;
+    }
+
+    /**
+     * Delete a existing {@link Mentee}
+     *
+     * @param id which is the identifier of the {@link Mentee}
+     * @return {@code true} if {@link Mentee} gets deleted
+     * @throws ResourceNotFoundException if {@link Mentee} for {@code id} doesn't exist
+     */
+    public boolean deleteMentee(int id)
```

Comment:
Make the return type `void` (because the method will never return `false`)

Reply:
Done